serverlessworkflow/modules/ROOT/pages/data-index/data-index-service.adoc

```diff
@@ -0,0 +1,375 @@
+              value: postgresql
+            - name: QUARKUS_HIBERNATE_ORM_DATABASE_GENERATION
+              value: update
+            - name: QUARKUS_KAFKA_HEALTH_ENABLE
+              value: "false"
+            - name: QUARKUS_HTTP_CORS
+              value: "true"
+            - name: QUARKUS_HTTP_CORS_ORIGINS
+              value: /.*/
+            - name: QUARKUS_FLYWAY_MIGRATE_AT_START <4>
+              value: "true"
+            - name: KOGITO_DATA_INDEX_QUARKUS_PROFILE <3>
+              value: "http-events-support"
+            - name: QUARKUS_HTTP_PORT
+              value: "8080"
+---
+apiVersion: eventing.knative.dev/v1
+kind: Trigger <5>
+metadata:
+  name: data-index-service-postgresql-processes-trigger
+spec:
+  broker: default
+  filter:
+    attributes:
+      type: ProcessInstanceEvent <6>
+  subscriber:
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: data-index-service-postgresql
+    uri: /processes <7>
+---
+apiVersion: eventing.knative.dev/v1
+kind: Trigger <5>
+metadata:
+  name: data-index-service-postgresql-jobs-trigger
+spec:
+  broker: default
+  filter:
+    attributes:
+      type: JobEvent <6>
+  subscriber:
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: data-index-service-postgresql
+    uri: /jobs <7>
+----
+<1> Reference the right {data_index_ref} image to match with the type of Database, in this case `quay.io/kiegroup/kogito-data-index-postgresql:latest`
+<2> Provide the database connection properties
+<3> KOGITO_DATA_INDEX_QUARKUS_PROFILE: http-events-support to use the http-connector.
+<4> To initialize the database schema at start using flyway
+<5> Trigger definition to filter the events that arrive to the Sink and pass them to {data_index_ref}
+<6> Type of event to filter
+<7> The URI where the {data_index_ref} service is expecting to consume those types of events.
+
+This deployment is using KOGITO_DATA_INDEX_QUARKUS_PROFILE: http-events-support. Workflow applications need to configure the connector to use `quarkus-http` and send the events to the Knative `K_SINK`:
+[source,properties]
+----
+mp.messaging.outgoing.kogito-processinstances-events.connector=quarkus-http
+mp.messaging.outgoing.kogito-processinstances-events.url=${K_SINK}
+mp.messaging.outgoing.kogito-processinstances-events.method=POST
+----
+
+[NOTE]
+====
+You can find more information about Knative eventing and `K_SINK` environment variable in xref:eventing/consume-produce-events-with-knative-eventing.adoc[]
+====
+
+A full example where the {data_index_ref} service standalone is deployed using Knative eventing can be found as part of xref:use-cases/timeout-showcase-example.adoc#execute-quarkus-project-standalone-services[Quarkus Workflow Project with standalone services] guide.
+
+When this {data_index_ref} service is deployed, a link:{kubernetes_configmap_url}[`configMap`] resource is created for the `application.properties` configuration of the {data_index_ref} service.
```

Comment:
```suggestion
When this {data_index_ref} service is deployed, a link:{kubernetes_configmap_url}[`configMap`] resource is created for the `application.properties` configuration of the {data_index_ref} service.
```

Now I see the path :)

But what is creating the configmap ? I mean Data Index is not managed by the Kogito Serverless Operator.
So unless you created the ConfigMap in the above example yaml, it does not exist and is not link to the Deployment ...

I would say the best way to add properties to the Data Index for now is via env vars.